packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1396,7 +1396,7 @@ async function handler(
         safeStringify({ error, results })
       );
     } else {
-      const metadata: AdditionalInformation = {};
+      const additionalInformation: AdditionalInformation = {};
```

Comment:
Better name, avoids confusion with another metadata variable in the function but in different scope.